rasa/server.py

```diff
@@ -476,6 +479,124 @@ async def hello(request: Request):
         return response.text("Hello from Rasa: " + rasa.__version__)
 
 
+def async_if_callback_url(f: Callable[..., Coroutine]) -> Callable:
```

Comment:
We don't have any equivalent of integration tests for rasa OSS where we could test these right?

Reply:
Nope. However, I added a test unit test for the cross-validation which mocks the callback URL and checks if the request was done as expected.